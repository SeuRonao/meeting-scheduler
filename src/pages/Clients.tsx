import AddClient from "../components/AddClient/AddClient";
import ClientTable from "../components/ClientTable/ClientTable";

export default function Clients() {
  return (
    <>
      <h1 className="text-center my-3">Clients</h1>
      <AddClient />
      <ClientTable />
    </>
  );
}
