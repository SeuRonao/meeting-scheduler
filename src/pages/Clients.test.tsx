import { create, act, ReactTestRenderer } from "react-test-renderer";
import Clients from "./Clients";

describe("Testing Clients", () => {
  test("renders correctly", async () => {
    let component: ReactTestRenderer;
    await act(async () => {
      component = create(<Clients />);
    });
    expect(component!.toJSON()).toMatchSnapshot();
  });
});
