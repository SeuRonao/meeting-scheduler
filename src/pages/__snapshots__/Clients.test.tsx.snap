// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Clients renders correctly 1`] = `
Array [
  <h1
    className="text-center my-3"
  >
    title
  </h1>,
  <button
    className="btn btn-primary"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    new-client-button
  </button>,
]
`;
